### ISSUE TITLE
Inline disable/enable linters are being ignored

### ISSUE THREAD

**Issue by peavers** [brigade/scss-lint#448]
Was working fine a couple of weeks back, we have since updated our dev environments and inline linters seem to be ignored 

```CSS
// scss-lint:disable PropertySortOrder
.page-wrapper {
  margin: 0 auto -200;
  min-height: 100%;
  height: auto;
}
// scss-lint:enable PropertySortOrder
```

Still returns:

```
PropertySortOrder:  Properties should be sorted according to the custom order specified by the configuration
```

I note the return message is now "according to the custom order..." whereas it used to tell us the custom order.

**Comment by sds** [comment]
Hey @peavers, thanks for the report.

I can't reproduce this with the latest version of `scss-lint`. Running `scss-lint` 0.37.0 (latest version at time of writing) against the snippet you provided results in no lints reported.

However, it's possible that this is related to #429 which is fixed on `master`, but as I already mentioned I can't reproduce with version 0.37.0, which doesn't have that fix.

If you have any other information to provide, that would be extremely helpful; otherwise I'm going to have to close due to lack of reproducibility.

**Comment by peavers** [comment]
I just rebuilt the server again and can confirm I'm no longer having any issues so it must have been something unique to me, or I just incorrectly configured something at one point. 

Confirmed working correctly with scss-lint 0.38.0
